src/main/java/com/github/tomakehurst/wiremock/admin/model/SnapshotStubMappingPostProcessor.java

```diff
@@ -0,0 +1,49 @@
+package com.github.tomakehurst.wiremock.admin.model;
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+/**
+ * Transform stub mappings using any applicable StubMappingTransformers and extract response body when applicable
+ */
+public class SnapshotStubMappingPostProcessor {
```

Comment:
I haven't added unit tests for this class yet. I'm going to wait until we have the details ironed out first.